content/purgatorio/collect-and-prepare-data/data-collection.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Data Collection
+author: neomatrix369
+description: The purpose of this guide is to talk about data collection and the different steps you need to do to prepare your dataset even before we can think of Machine Learning models and the works.
+---
+
+# What you will learn 
```

Comment:
Test Comment. Is this the right way to post a comment?

Reply:
Yes it is! You have used GitHub before right? And done reviews of PRs? ;)

Reply:
Actually it's the first time I post comments inside the files (the other times I was answering yours), so yes :D

Reply:
Oh! 😮